### ISSUE TITLE
SubGrid.GetSubGridItemsCount

### ISSUE THREAD

**Issue by rngarad** [Microsoft/EasyRepro#1379]
The xrmApp.Entity.SubGrid.GetSubGridItemsCount returning **0** value even subgrid having items 
Code to reproduce:
 LandingCOuntriesCount = xrmApp.Entity.SubGrid.GetSubGridItemsCount("LandingCountries");
            Assert.AreEqual(p0, LandingCOuntriesCount);

**Comment by krishnacs1993** [comment]
Yes, even I am facing the same issue. GetSubGridItemsCount is returning 0 even if there are records in the subgrid.
